app/views/archives/index.html.erb

```diff
@@ -11,7 +11,7 @@
 </header>
 
 <div class="h-feed">
-  <% @articles.sort.reverse.each do |year, months| %>
+  <% @archive.sort.reverse.each do |year, months| %>
```

Comment:
Not sure why I had the `.sort.reverse` here in the view. But it can probably get moved into the controller.